### ISSUE TITLE
[BUG]: Encountering the 'browserContext.newPage: Target closed' error when using web-test-runner after macOS 14, specifically failing on WebKit

### ISSUE THREAD

**Issue by erbilnas** [Trendyol/baklava#780]
### Issue description

Related issue: https://github.com/modernweb-dev/web/issues/2589

```
$ npm run test
```

```
> @trendyol/baklava@0.0.0-dev test
> web-test-runner --coverage
```

```
src/utilities/event.test.ts:

 ❌ browserContext.newPage: Target closed  (failed on Webkit)
    browserContext.newPage: Target closed
        at PlaywrightLauncher.createPage [as createPageFn] (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/index.js:31:155)
        at PlaywrightLauncher.createNewPage (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/PlaywrightLauncher.js:68:43)
        at async PlaywrightLauncher.startSession (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/PlaywrightLauncher.js:42:22)

src/utilities/asset-paths.test.ts:

 ❌ browserContext.newPage: Target closed  (failed on Webkit)
    browserContext.newPage: Target closed
        at PlaywrightLauncher.createPage [as createPageFn] (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/index.js:31:155)
        at PlaywrightLauncher.createNewPage (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/PlaywrightLauncher.js:68:43)
        at async PlaywrightLauncher.startSession (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/PlaywrightLauncher.js:42:22)

src/utilities/form-control.test.ts:

 ❌ browserContext.newPage: Target closed  (failed on Webkit)
    browserContext.newPage: Target closed
        at PlaywrightLauncher.createPage [as createPageFn] (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/index.js:31:155)
        at PlaywrightLauncher.createNewPage (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/PlaywrightLauncher.js:68:43)
        at async PlaywrightLauncher.startSession (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/PlaywrightLauncher.js:42:22)

src/components/alert/bl-alert.test.ts:

 ❌ browserContext.newPage: Target closed  (failed on Webkit)
    browserContext.newPage: Target closed
        at PlaywrightLauncher.createPage [as createPageFn] (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/index.js:31:155)
        at PlaywrightLauncher.createNewPage (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/PlaywrightLauncher.js:68:43)
        at async PlaywrightLauncher.startSession (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/PlaywrightLauncher.js:42:22)

src/utilities/string-boolean.converter.test.ts:

 ❌ browserContext.newPage: Target closed  (failed on Webkit)
    browserContext.newPage: Target closed
        at PlaywrightLauncher.createPage [as createPageFn] (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/index.js:31:155)
        at PlaywrightLauncher.createNewPage (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/PlaywrightLauncher.js:68:43)
        at async PlaywrightLauncher.startSession (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/PlaywrightLauncher.js:42:22)

src/components/badge/bl-badge.test.ts:

 ❌ browserContext.newPage: Target closed  (failed on Webkit)
    browserContext.newPage: Target closed
        at PlaywrightLauncher.createPage [as createPageFn] (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/index.js:31:155)
        at PlaywrightLauncher.createNewPage (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/PlaywrightLauncher.js:68:43)
        at async PlaywrightLauncher.startSession (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/PlaywrightLauncher.js:42:22)

src/components/button/bl-button.test.ts:

 ❌ browserContext.newPage: Target closed  (failed on Webkit)
    browserContext.newPage: Target closed
        at PlaywrightLauncher.createPage [as createPageFn] (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/index.js:31:155)
        at PlaywrightLauncher.createNewPage (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/PlaywrightLauncher.js:68:43)
        at async PlaywrightLauncher.startSession (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/PlaywrightLauncher.js:42:22)

src/components/checkbox-group/bl-checkbox-group.test.ts:

 ❌ browserContext.newPage: Target closed  (failed on Webkit)
    browserContext.newPage: Target closed
        at PlaywrightLauncher.createPage [as createPageFn] (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/index.js:31:155)
        at PlaywrightLauncher.createNewPage (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/PlaywrightLauncher.js:68:43)
        at async PlaywrightLauncher.startSession (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/PlaywrightLauncher.js:42:22)

src/components/dialog/bl-dialog.test.ts:

 ❌ browserContext.newPage: Target closed  (failed on Webkit)
    browserContext.newPage: Target closed
        at PlaywrightLauncher.createPage [as createPageFn] (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/index.js:31:155)
        at PlaywrightLauncher.createNewPage (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/PlaywrightLauncher.js:68:43)
        at async PlaywrightLauncher.startSession (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/PlaywrightLauncher.js:42:22)

src/components/drawer/bl-drawer.test.ts:

 ❌ browserContext.newPage: Target closed  (failed on Webkit)
    browserContext.newPage: Target closed
        at PlaywrightLauncher.createPage [as createPageFn] (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/index.js:31:155)
        at PlaywrightLauncher.createNewPage (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/PlaywrightLauncher.js:68:43)
        at async PlaywrightLauncher.startSession (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/PlaywrightLauncher.js:42:22)

src/components/dropdown/bl-dropdown.test.ts:

 ❌ browserContext.newPage: Target closed  (failed on Webkit)
    browserContext.newPage: Target closed
        at PlaywrightLauncher.createPage [as createPageFn] (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/index.js:31:155)
        at PlaywrightLauncher.createNewPage (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/PlaywrightLauncher.js:68:43)
        at async PlaywrightLauncher.startSession (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/PlaywrightLauncher.js:42:22)

src/components/icon/bl-icon.test.ts:

 ❌ browserContext.newPage: Target closed  (failed on Webkit)
    browserContext.newPage: Target closed
        at PlaywrightLauncher.createPage [as createPageFn] (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/index.js:31:155)
        at PlaywrightLauncher.createNewPage (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/PlaywrightLauncher.js:68:43)
        at async PlaywrightLauncher.startSession (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/PlaywrightLauncher.js:42:22)

src/components/input/bl-input.test.ts:

 ❌ browserContext.newPage: Target closed  (failed on Webkit)
    browserContext.newPage: Target closed
        at PlaywrightLauncher.createPage [as createPageFn] (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/index.js:31:155)
        at PlaywrightLauncher.createNewPage (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/PlaywrightLauncher.js:68:43)
        at async PlaywrightLauncher.startSession (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/PlaywrightLauncher.js:42:22)

src/components/notification/bl-notification.test.ts:

 ❌ browserContext.newPage: Target closed  (failed on Webkit)
    browserContext.newPage: Target closed
        at PlaywrightLauncher.createPage [as createPageFn] (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/index.js:31:155)
        at PlaywrightLauncher.createNewPage (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/PlaywrightLauncher.js:68:43)
        at async PlaywrightLauncher.startSession (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/PlaywrightLauncher.js:42:22)

src/components/pagination/bl-pagination.test.ts:

 ❌ browserContext.newPage: Target closed  (failed on Webkit)
    browserContext.newPage: Target closed
        at PlaywrightLauncher.createPage [as createPageFn] (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/index.js:31:155)
        at PlaywrightLauncher.createNewPage (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/PlaywrightLauncher.js:68:43)
        at async PlaywrightLauncher.startSession (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/PlaywrightLauncher.js:42:22)

src/components/popover/bl-popover.test.ts:

 🚧 Browser logs on Chromium:
      BlPopover target only accepts an Element instance or a string id of a DOM element.
 🚧 Browser logs on Firefox:
      BlPopover target only accepts an Element instance or a string id of a DOM element.

 ❌ browserContext.newPage: Target closed  (failed on Webkit)
    browserContext.newPage: Target closed
        at PlaywrightLauncher.createPage [as createPageFn] (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/index.js:31:155)
        at PlaywrightLauncher.createNewPage (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/PlaywrightLauncher.js:68:43)
        at async PlaywrightLauncher.startSession (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/PlaywrightLauncher.js:42:22)

src/components/progress-indicator/bl-progress-indicator.test.ts:

 ❌ browserContext.newPage: Target closed  (failed on Webkit)
    browserContext.newPage: Target closed
        at PlaywrightLauncher.createPage [as createPageFn] (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/index.js:31:155)
        at PlaywrightLauncher.createNewPage (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/PlaywrightLauncher.js:68:43)
        at async PlaywrightLauncher.startSession (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/PlaywrightLauncher.js:42:22)

src/components/radio-group/bl-radio-group.test.ts:

 ❌ browserContext.newPage: Target closed  (failed on Webkit)
    browserContext.newPage: Target closed
        at PlaywrightLauncher.createPage [as createPageFn] (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/index.js:31:155)
        at PlaywrightLauncher.createNewPage (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/PlaywrightLauncher.js:68:43)
        at async PlaywrightLauncher.startSession (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/PlaywrightLauncher.js:42:22)

src/components/select/bl-select.test.ts:

 ❌ browserContext.newPage: Target closed  (failed on Webkit)
    browserContext.newPage: Target closed
        at PlaywrightLauncher.createPage [as createPageFn] (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/index.js:31:155)
        at PlaywrightLauncher.createNewPage (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/PlaywrightLauncher.js:68:43)
        at async PlaywrightLauncher.startSession (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/PlaywrightLauncher.js:42:22)

src/components/switch/bl-switch.test.ts:

 ❌ browserContext.newPage: Target closed  (failed on Webkit)
    browserContext.newPage: Target closed
        at PlaywrightLauncher.createPage [as createPageFn] (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/index.js:31:155)
        at PlaywrightLauncher.createNewPage (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/PlaywrightLauncher.js:68:43)
        at async PlaywrightLauncher.startSession (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/PlaywrightLauncher.js:42:22)

src/components/tab-group/bl-tab-group.test.ts:

 ❌ browserContext.newPage: Target closed  (failed on Webkit)
    browserContext.newPage: Target closed
        at PlaywrightLauncher.createPage [as createPageFn] (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/index.js:31:155)
        at PlaywrightLauncher.createNewPage (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/PlaywrightLauncher.js:68:43)
        at async PlaywrightLauncher.startSession (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/PlaywrightLauncher.js:42:22)

src/components/textarea/bl-textarea.test.ts:

 ❌ browserContext.newPage: Target closed  (failed on Webkit)
    browserContext.newPage: Target closed
        at PlaywrightLauncher.createPage [as createPageFn] (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/index.js:31:155)
        at PlaywrightLauncher.createNewPage (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/PlaywrightLauncher.js:68:43)
        at async PlaywrightLauncher.startSession (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/PlaywrightLauncher.js:42:22)

src/components/tooltip/bl-tooltip.test.ts:

 🚧 Browser logs on Chromium:
      BlPopover target only accepts an Element instance or a string id of a DOM element.
      BlPopover target only accepts an Element instance or a string id of a DOM element.
      BlPopover target only accepts an Element instance or a string id of a DOM element.
      BlPopover target only accepts an Element instance or a string id of a DOM element.
      BlPopover target only accepts an Element instance or a string id of a DOM element.
      BlPopover target only accepts an Element instance or a string id of a DOM element.
      BlPopover target only accepts an Element instance or a string id of a DOM element.
      BlPopover target only accepts an Element instance or a string id of a DOM element.
      BlPopover target only accepts an Element instance or a string id of a DOM element.
      BlPopover target only accepts an Element instance or a string id of a DOM element.
 🚧 Browser logs on Firefox:
      BlPopover target only accepts an Element instance or a string id of a DOM element.
      BlPopover target only accepts an Element instance or a string id of a DOM element.
      BlPopover target only accepts an Element instance or a string id of a DOM element.
      BlPopover target only accepts an Element instance or a string id of a DOM element.
      BlPopover target only accepts an Element instance or a string id of a DOM element.
      BlPopover target only accepts an Element instance or a string id of a DOM element.
      BlPopover target only accepts an Element instance or a string id of a DOM element.
      BlPopover target only accepts an Element instance or a string id of a DOM element.
      BlPopover target only accepts an Element instance or a string id of a DOM element.
      BlPopover target only accepts an Element instance or a string id of a DOM element.

 ❌ browserContext.newPage: Target closed  (failed on Webkit)
    browserContext.newPage: Target closed
        at PlaywrightLauncher.createPage [as createPageFn] (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/index.js:31:155)
        at PlaywrightLauncher.createNewPage (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/PlaywrightLauncher.js:68:43)
        at async PlaywrightLauncher.startSession (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/PlaywrightLauncher.js:42:22)

src/components/checkbox-group/checkbox/bl-checkbox.test.ts:

 ❌ browserContext.newPage: Target closed  (failed on Webkit)
    browserContext.newPage: Target closed
        at PlaywrightLauncher.createPage [as createPageFn] (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/index.js:31:155)
        at PlaywrightLauncher.createNewPage (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/PlaywrightLauncher.js:68:43)
        at async PlaywrightLauncher.startSession (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/PlaywrightLauncher.js:42:22)

src/components/dropdown/group/bl-dropdown-group.test.ts:

 ❌ browserContext.newPage: Target closed  (failed on Webkit)
    browserContext.newPage: Target closed
        at PlaywrightLauncher.createPage [as createPageFn] (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/index.js:31:155)
        at PlaywrightLauncher.createNewPage (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/PlaywrightLauncher.js:68:43)
        at async PlaywrightLauncher.startSession (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/PlaywrightLauncher.js:42:22)

src/components/dropdown/item/bl-dropdown-item.test.ts:

 🚧 Browser logs on Chromium:
      bl-dropdown-item is designed to be used inside a bl-dropdown-group or bl-dropdown BlDropdownItem: <bl-dropdown-item></bl-dropdown-item>
      bl-dropdown-item is designed to be used inside a bl-dropdown-group or bl-dropdown BlDropdownItem: <bl-dropdown-item icon="info"></bl-dropdown-item>
      bl-dropdown-item is designed to be used inside a bl-dropdown-group or bl-dropdown BlDropdownItem: <bl-dropdown-item>dropdown-item</bl-dropdown-item>
 🚧 Browser logs on Firefox:
      bl-dropdown-item is designed to be used inside a bl-dropdown-group or bl-dropdown BlDropdownItem: <bl-dropdown-item></bl-dropdown-item>
      bl-dropdown-item is designed to be used inside a bl-dropdown-group or bl-dropdown BlDropdownItem: <bl-dropdown-item icon="info"></bl-dropdown-item>
      bl-dropdown-item is designed to be used inside a bl-dropdown-group or bl-dropdown BlDropdownItem: <bl-dropdown-item>dropdown-item</bl-dropdown-item>

 ❌ browserContext.newPage: Target closed  (failed on Webkit)
    browserContext.newPage: Target closed
        at PlaywrightLauncher.createPage [as createPageFn] (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/index.js:31:155)
        at PlaywrightLauncher.createNewPage (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/PlaywrightLauncher.js:68:43)
        at async PlaywrightLauncher.startSession (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/PlaywrightLauncher.js:42:22)

src/components/notification/card/bl-notification-card.test.ts:

 ❌ browserContext.newPage: Target closed  (failed on Webkit)
    browserContext.newPage: Target closed
        at PlaywrightLauncher.createPage [as createPageFn] (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/index.js:31:155)
        at PlaywrightLauncher.createNewPage (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/PlaywrightLauncher.js:68:43)
        at async PlaywrightLauncher.startSession (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/PlaywrightLauncher.js:42:22)

src/components/radio-group/radio/bl-radio.test.ts:

 🚧 Browser logs on Chromium:
      bl-radio is designed to be used inside a bl-radio-group BlRadio: <bl-radio value="cc">Credit Card</bl-radio>
 🚧 Browser logs on Firefox:
      bl-radio is designed to be used inside a bl-radio-group BlRadio: <bl-radio value="cc">Credit Card</bl-radio>

 ❌ browserContext.newPage: Target closed  (failed on Webkit)
    browserContext.newPage: Target closed
        at PlaywrightLauncher.createPage [as createPageFn] (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/index.js:31:155)
        at PlaywrightLauncher.createNewPage (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/PlaywrightLauncher.js:68:43)
        at async PlaywrightLauncher.startSession (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/PlaywrightLauncher.js:42:22)

src/components/select/option/bl-select-option.test.ts:

 ❌ browserContext.newPage: Target closed  (failed on Webkit)
    browserContext.newPage: Target closed
        at PlaywrightLauncher.createPage [as createPageFn] (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/index.js:31:155)
        at PlaywrightLauncher.createNewPage (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/PlaywrightLauncher.js:68:43)
        at async PlaywrightLauncher.startSession (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/PlaywrightLauncher.js:42:22)

src/components/tab-group/tab/bl-tab.test.ts:

 ❌ browserContext.newPage: Target closed  (failed on Webkit)
    browserContext.newPage: Target closed
        at PlaywrightLauncher.createPage [as createPageFn] (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/index.js:31:155)
        at PlaywrightLauncher.createNewPage (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/PlaywrightLauncher.js:68:43)
        at async PlaywrightLauncher.startSession (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/PlaywrightLauncher.js:42:22)

src/components/tab-group/tab-panel/bl-tab-panel.test.ts:

 ❌ browserContext.newPage: Target closed  (failed on Webkit)
    browserContext.newPage: Target closed
        at PlaywrightLauncher.createPage [as createPageFn] (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/index.js:31:155)
        at PlaywrightLauncher.createNewPage (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/PlaywrightLauncher.js:68:43)
        at async PlaywrightLauncher.startSession (/Users/erbilnas/Documents/Projects/baklava/node_modules/@web/test-runner-playwright/dist/PlaywrightLauncher.js:42:22)
```

```
Chromium: |██████████████████████████████| 31/31 test files | 351 passed, 0 failed, 9 skipped
Firefox:  |██████████████████████████████| 31/31 test files | 351 passed, 0 failed, 9 skipped
Webkit:   |██████████████████████████████| 31/31 test files | 0 passed, 0 failed

Code coverage: 100 %
View full coverage report at coverage/lcov-report/index.html

Error while running tests.
```

### Baklava Version

2.3.0

### Operating system

- OS: macOS 14.2 (23C64)
- Browser: Microsoft Edge Version 120.0.2210.144 (Official build) (arm64)

- VS Code
```
Version: 1.85.2
Commit: 8b3775030ed1a69b13e4f4c628c612102e30a681
Date: 2024-01-18T06:40:11.430Z
Electron: 25.9.7
ElectronBuildId: 26354273
Chromium: 114.0.5735.289
Node.js: 18.15.0
V8: 11.4.183.29-electron.0
OS: Darwin arm64 23.2.0
```

- Package
```
"@web/dev-server-esbuild": "0.2.16",
"@web/dev-server-import-maps": "^0.0.7",
"@web/dev-server-rollup": "^0.3.17",
"@web/test-runner": "^0.15.2",
"@web/test-runner-playwright": "^0.9.0",
"@web/test-runner-puppeteer": "^0.12.0",
```

### Priority this issue should have

High (immediate attention needed)

### Please review the checkboxes that are applicable.

- [ ] This issue is directly related to React

**Comment by AykutSarac** [comment]
Is this still valid?

**Comment by erbilnas** [comment]
yes

**Comment by umitkucuk** [comment]
Same here, any updates on this? @AykutSarac 

node: v18.18.1
npm: v9.8.1

**Comment by erbilnas** [comment]
Fixed on #969
